code/modules/mob/living/simple_animal/friendly/snake.dm

```diff
@@ -58,9 +66,67 @@
-                visible_message("<span class='notice'>[name] consumes [target] in a single gulp!</span>", "<span class='notice'>You consume [target] in a single gulp!</span>")
-                QDEL_NULL(target)
-                adjustBruteLoss(-2)
-        else
-                return ..()
+		if(istype(target, /mob/living/simple_animal/mouse) && melee_damage_upper > 0) //asclepius's snakes are pacifists
+				visible_message("<span class='notice'>[name] consumes [target] in a single gulp!</span>", "<span class='notice'>You consume [target] in a single gulp!</span>")
+				QDEL_NULL(target)
+				adjustBruteLoss(-2)
+		else
+				return ..()
+
+/mob/living/simple_animal/hostile/retaliate/poison/snake/asclepius
+	name = "Asclepius's snake"
+	desc = "A mystical snake previously trapped upon the Rod of Asclepius, now freed of its burden. Its bites are rumored to have healing properties."
+	poison_type = /datum/reagent/medicine/omnizine //while using godblood instead of omnizine here would be flavorful, doing that would also allow the snake to pump you with up to 150u of godblood (it has an OD threshold of 150u), which would basically leave you set for the rest of the round
+	gold_core_spawnable = NO_SPAWN //only obtainable from the rod of ascelpius and/or adminbus
+	melee_damage_lower = 0 //do no harm
+	melee_damage_upper = 0
+	friendly_verb_continuous = "bites" //so that it will bite people instead of nuzzle them
+	friendly_verb_simple = "bite"
+	buffed = FALSE //so that you can't use a wumborian fugu gland on it to allow it to break its oath
+	food_type = list(/obj/item/reagent_containers/food/snacks/grown) //it's vegan (or at least trying to be a vegetarian)
+	careful = TRUE //otherwise, NPC asclepius snakes will kill people by ODing them on omnizine
+	attack_same = TRUE //we "attack" our allies to heal them
+
+/mob/living/simple_animal/hostile/retaliate/poison/snake/asclepius/Initialize()
+	. = ..()
+	var/datum/atom_hud/H = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	H.add_hud_to(src)
+	notify_ghosts("A controllable snake of Asclepius has been created in \the [get_area(src)].", source = src, action = NOTIFY_ORBIT, flashwindow = FALSE, header = "Sentient Snake Created")
+
+/mob/living/simple_animal/hostile/retaliate/poison/snake/asclepius/ListTargets(atom/the_target)
+	var/list/patients = list()
+	for (var/mob/living/living_in_oview in oview(vision_range, targets_from)) //get list of things in vision range
+		//can I heal you?
+		if(!living_in_oview.reagents || (living_in_oview.health >= living_in_oview.maxHealth)) //yeah, clone damage and such will make the snake be obsessed with one target, but eh, even divine doctors ain't perfect
+			continue
+		if(careful && poison_type.overdose_threshold && living_in_oview.reagents.has_reagent(poison_type, (poison_type.overdose_threshold - poison_per_bite))) //if they're already full of our reagent, let's not remain obsessed with them
+			continue
+	patients += living_in_oview
```

Comment:
It isn't a thing here man, you need to indent it a level down

Reply:
thanks, missed that one